index.html

```diff
@@ -1,75 +1,110 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
-  <meta charset="utf-8">
+    <meta charset="utf-8">
 
-  <title>Sprint Challenge - Home</title>
+    <title>Sprint Challenge - Home</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
 
 </head>
 
 <body>
     <div class="container">
-   
+        <header>
+            <section class="logo">
+                <section class="logo-image">
```

Comment:
Always try to use alt="" attributes when using images for accessibility to the site.